docs/learn/using-zeitgeist-markets.md

```diff
@@ -300,49 +171,97 @@ dispute for the token which the market eventually resolved to receive their
+market is open (exactly one of each outcome token from the market) at the exact
+price of 1 ZTG (plus transaction fee). The ZTG paid for the mint is placed in
+the market's _prize pool_. _All outcome tokens are created by minting them in
+this fashion._ Full sets may also be destroyed. For every full set destroyed the
+user receives 1 ZTG back from the prize pool.
+
+When a market is created, the prize pool is empty, and the balance of the prize
+pool cannot be changed except by minting and burning full sets. These rules
+guarantee the prize pool contains exactly 1 ZTG for every full set of outcome
+tokens in circulation. This was, the winning outcome token is backed 1:1 in ZTG.
+
+The prize pool should not be confused with the market's liquidity pool, which is
+described further below.
+
+For example, Alice has 3.7 ZTG in her wallet. She mints 3.5 full sets for the
+James Webb Space Telescope market, pays 3.5 ZTG (which goes into the prize pool
+of the JWST market) plus transaction fees, and receives 3.5 JWSTYES and 3.5
+JWSTNO.
+
+Bob, on the other hand, has 2.1 JWSTYES and 3.4 JWSTNO tokens. He decides to
+destroy 2.1 full sets (leaving him with only 1.3 JWSTNO) and receive 2.1 ZTG
+back from the prize pool. Note that Bob could not have destroyed any more full
+sets, as he owns no more JWSTYES.
+
+### The Liquidity Pool
 
-### Arbitrage on Zeitgeist
+Zeitgeist supports a constant mean market maker, also referred to as constant
+product market maker or CPMM on our platform, which is based on the
+[Balancer AMM](https://balancer.fi/whitepaper.pdf), a variation on the basic
+$x \cdot y = \mathrm{const}$ formula which allows different assets to have
+different _weights_, which define their impact on price.
 
-Zeitgeist's AMM, the Rikiddo scoring rule, does not guarantee that for any
-market the sum of the prices of all outcome assets is approximately equal to 1
-ZTG, or that the price of any outcome tokens remains below 1 ZTG. High prices
-are indicators of low liquidity or a volatile market.
+This AMM's liquidity pool contains balances of ZTG and of all outcome tokens of
+the market. Users can trade their tokens with tokens stored in the pool: They
+can buy outcome tokens from the pool with ZTG which is then added to the pool,
+or sell outcome tokens to the pool for some of the pool's ZTG.
 
-This creates opportunities for users to add or remove liquidity to profit from
-arbitrage:
+By default, a new market has no liquidity pool. Instead, the pool must either be
+deployed by the market creator, or by some external liquidity provider. After
+the pool is created, others may _join_ the liquidity pool by providing
+additional liquidity. When deploying liquidity into a pool, a liquidity provider
+will usually provide the same amount of full sets of outcome tokens as ZTG ($x$
+of each outcome token and $x$ ZTG). The current minimum for $x$ is 100, making a
+total value of 200 ZTG.
```

Comment:
Will neo-swaps allow an automatic opening of the liquidity pool in the future like we had with CPMM? Because currently the market creator has to open the market with their liquidity at the creation time, right?

Reply:
Not sure if that's something that we actually want (as discussed privately). Seems like a lot of work for something that isn't really our responsibility.